app/src/main/java/org/oppia/android/app/settings/profile/ProfileResetPinViewModel.kt

```diff
@@ -8,21 +8,21 @@ import javax.inject.Inject
 /** The ViewModel for [ProfileResetPinActivity]. */
 @ActivityScope
 class ProfileResetPinViewModel @Inject constructor() : ObservableViewModel() {
-  /** Variable for storing error message in first text input layout. */
+  /** The error message is shown if the second pin not matches with the first pin. */
```

Comment:
```suggestion
  /** The error message shown if the second pin not matches with the first pin. */
```

Ditto elsewhere. The docstring should describe 'what' rather than being a standard sentence (the first bit of a KDoc is generally always a sentence fragment).

Reply:
done